packages/lib-ts/src/types/ByteArray.ts

```diff
@@ -303,6 +314,13 @@ export class ByteArray extends Uint8Array implements Serializable {
     return bytesToHexString(this)
   }
 
+  /**
+   * Converts this ByteArray to a base58 string.
+   */
+  toBase58String(): string {
+    return bytesToBase58String(this)
```

Comment:
Why do you use a helper function here and then in `fromBase58String` you directly use the decode from the library. 

I like using the helpers functions and that the library usage is only in the helpers file

Reply:
I copied the pattern from the existing method `toHexString`, where it uses a helper function.

If this should be otherwise, let me know and I can refactor both methods. But since the helpers (more precisely `bytesToHexString`) are already commonly being used this would mean repeating a bunch of code across several files.

Otherwise, I can just refactor this one, but then it would be inconsistent in style. I'm open to suggestions here, I'll just follow any guidelines that you'd like as for code style.

Reply:
I'm good with `bytesToBase58String`. I'm saying that instead of using the [decode](https://github.com/mimic-protocol/tooling/pull/108/files/8f41376ac9d510c6962dc19f1d9f2473c9a44cdf#diff-bd78d09f29848a30261312f6844ed17951aeb76732e226aa231e1a37821d5947R119) we should create a new function `base58StringToBytes` in helpers and use it